### PR TITLE
Add a general infinite scroll feature

### LINKED ISSUE #861: Autoload contribution activities on scroll down
This is already done for the newsfeed. It'd be nice if this was also made available for the contribution activities on user profiles.

### PR DESCRIPTION
**Pull request by DerTieran**
This also adds infinit scroll to the contribution activities and replaces the news feed auto loading.

Instead of just copying the whole code like in #1505 this generalizes the auto load feature as suggested by @bfred-it.

- [x] I still need to add the migration but I need the PR number for that. :smile: 
- [x] It also needs an `readme` update, does that also need to mention the migration?

Closes #861

### CODE REVIEW

**Comment by DerTieran** [comment]
I would also capitalize the `loading...` text, because the normal button text is also capitalized.

This isn't a big issue for me, but I think it would fit into this PR :smile:

File: readme.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ GitHub Enterprise is also supported. More info in the options.
 - [Approve or reject reviews faster with one-click review type buttons.](https://user-images.githubusercontent.com/1402241/34326942-529cb7c0-e8f3-11e7-9bee-98b667e18a90.png)
 - Pressing `Cancel` on an inline comment opens a prompt to prevent accidental cancelling.
 - The `+` and `-` signs in diffs are made unselectable for easier copy-pasting.
-- The news feeds automagically expands when you scroll down.
+- The news feeds and contribution activities automagically expands when you scroll down.
```
Comment (bfred-it) [inline readme.md]: `s/expands/expand`

File: source/content.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ import addPrevNextButtonsToPRs from './features/prev-next-commit-buttons';
 import addFileCopyButton from './features/copy-file';
 // - import copyMarkdown from './features/copy-markdown';
 import linkifyCode from './features/linkify-urls-in-code';
-import autoLoadMoreNews from './features/auto-load-more-news';
+import infinitScroll from './features/infinit-scroll';
```
Comment (bfred-it) [inline source/content.js]: `s/infinit/infinite/` in all files

**Comment by sindresorhus** [comment]
Thanks, @DerTieran :)

**Pull request merged**